CONTRIBUTING.md

```diff
@@ -81,8 +81,9 @@ gems for your organization..**
 * Use Ruby 2.3.1
 * Use Rubygems 2.6.10
 * Install bundler: `gem install bundler`
-* Install Elastic Search: `brew install elasticsearch`
-  * Setup information: `brew info elasticsearch`
+* Install Elastic Search:
+  * ElasticSearch `1.*` is required. There isn't a homebrew package for `elasticsearch@1`, instead it's recommended to use [Docker](https://www.docker.com/)
+  * Pull ElasticSearch `1.7.6` from [hub.docker.com](https://hub.docker.com/_/elasticsearch/) and expose the container ports: `docker run -P elasticsearch:1.7.6`
```

Comment:
Sorry, i wasn't paying attention and only looked at the oldest version that was on the README. `1.5.2` can be pulled and works for me. I've updated the PR to use the correct version.